tests/contrib/operators/test_ecs_operator.py

```diff
@@ -133,6 +134,38 @@ def test_execute_without_failures(self, check_mock, wait_mock):
         self.assertEqual(self.ecs.arn,
                          'arn:aws:ecs:us-east-1:012345678910:task/d8c67b3c-ac87-4ffe-a847-4785bc3a8b55')
 
+    @mock.patch.object(ECSOperator, '_wait_for_task_ended')
+    @mock.patch.object(ECSOperator, '_check_success_task')
+    def test_fargate_run(self, check_mock, wait_mock):
```

Comment:
I think this test is almost the same as `test_execute_without_failures`. Could we use parameterized tests instead of writing two very similar tests, or would that make the code more confusing?

Reply:
Then `test_execute_without_failures` need to update somehow so:
```
  ecs_operator = ECSOperator(launch_type=launch_type, **self.ecs_operator_args)
  ecs_operator.execute(None)
  ...
  args = {
    'cluster': 'c',
    'launchType': launch_type,
    ...
  }
  if launch_type == 'FARGATE':
    args['platformVersion'] = 'LATEST'
  client_mock.run_task.assert_called_once_with(**args)
```
so I can do this if it dont confusing you

Reply:
@jmcarp Is there an example in the repo else where you can point at?